Formula/zlog.rb

```diff
@@ -0,0 +1,48 @@
+class Zlog < Formula
+  desc "High-performance C logging library"
+  homepage "https://github.com/HardySimpson/zlog"
+  url "https://github.com/HardySimpson/zlog/archive/1.2.12.tar.gz"
+  sha256 "9c6014a3f74d136c70255539beba11f30e1d3617d07ce7ea917b35f3e52bac20"
+
+  def install
+    system "make"
+    system "make", "PREFIX=#{prefix}", "install"
+  end
+
+  test do
+    (testpath/"zlog.conf").write <<~EOS
+      [formats]
+      simple = "%m%n"
+      [rules]
+      my_cat.DEBUG    >stdout; simple
+    EOS
+    (testpath/"zlog.c").write <<~EOS
+      #include <stdio.h>
+      #include <zlog.h>
+      int main() {
+        int rc;
+        zlog_category_t *c;
+
+        rc = zlog_init("zlog.conf");
+        if (rc) {
+          printf("init failed!");
+          return -1;
+        }
+
+        c = zlog_get_category("my_cat");
+        if (!c) {
+          printf("get cat failed!");
+          zlog_fini();
+          return -2;
+        }
+
+        zlog_info(c, "hello, zlog!");
+        zlog_fini();
+
+        return 0;
+      }
+    EOS
+    system ENV.cc, "zlog.c", "-L#{lib}", "-lzlog", "-lpthread", "-o", "zlog"
```

Comment:
It's a tiny tiny thing but for the sake of consistency can we stick to `"-o", "test"` and `system "./test"`, thanks. Just makes it easier to find all the instances at once when necessary/desirable.